### ISSUE TITLE
start() frame decode fatal error 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Khang-NT/aacdecoder-android#37]
```
1) I run the radio with this url http://159.253.145.180:7020/

2) After a few seconds my application closes

The server does not send the data when it is busy.

+ + + + + + +
curl http://159.253.145.180:7020/


ICY 400 Server Full
icy-notice1: <BR> SHOUTcast Distributed Network Audio Server / Linux v1.9.8 <BR>
icy-notice2: This server has reached its user limit <BR>
+ + + + +

The audio stream is not present and the decoder is in error:

06-24 15:51:29.365: ERROR / Decoder [OpenCORE-MP3] (19239): start () frame 
decode error = 11
06-24 15:51:29.365: ERROR / Decoder [OpenCORE-MP3] (19239): start () first 
frame can not be decoded - trying to sync again

How can I fix the problem?

```

Original issue reported on code.google.com by `fabri8bit` on 24 Jun 2013 at 2:04

**Comment by GoogleCodeExporter** [comment]
```
Fast fix work. ;)


```

Original comment by `fabri8bit` on 24 Jun 2013 at 2:58

Attachments:
- [mp3-opencore-decoder.c](https://storage.googleapis.com/google-code-attachments/aacdecoder-android/issue-37/comment-1/mp3-opencore-decoder.c)

**Comment by GoogleCodeExporter** [comment]
```
revision 39 - added check for the response code - both HTTP and shoutcast.
```

Original comment by `vbarta...@gmail.com` on 10 Jan 2014 at 7:49
- Changed state: **Started**
- Added labels: **Component-Java**

**Comment by GoogleCodeExporter** [comment]
Original comment by `vbarta...@gmail.com` on 10 Jan 2014 at 7:52

**Comment by GoogleCodeExporter** [comment]
```
included in version 0.8
```

Original comment by `vbarta...@gmail.com` on 12 Jan 2014 at 8:17
- Changed state: **Fixed**
